tests/js/session-tests.js

```diff
@@ -41,9 +40,10 @@ function node_require(module) {
     return require_method(module);
 }
 
-let fetch;
+let fetch, loginAdminUser;
 if (isNodeProcess) {
     fetch = node_require('node-fetch');
+    loginAdminUser = node_require('./admin-user-helper').loginAdminUser;
```

Comment:
admin-user-helper is already ran before each test. You can see how other tests are using it here
https://github.com/realm/realm-js/blob/master/tests/js/user-tests.js#L287-L296